### PR TITLE
Add position to Maps add method

### PR DESCRIPTION
**Pull request by gjmooney**
Add ``position`` parameter to Maps ``add`` method, allowing users to specify where in the layers stack they would add their new layer.

### CODE REVIEW

File: ipyleaflet/leaflet.py [hunk 1]
```diff
@@ -2676,13 +2676,15 @@ def __isub__(self, item):
     def __add__(self, item):
         return self.add(item)
 
-    def add(self, item):
+    def add(self, item, position=None):
```
Comment (martinRenou) [inline ipyleaflet/leaflet.py:2679]:
```suggestion
    def add(self, item, index=None):
```

**Review by martinRenou — COMMENTED** [review]
Looks like a nice addition! Would you be able to add some ui-tests for this? 

Also, I would rename `position` into `index`, what do you think?

**Comment by gjmooney** [comment]
I added the UI tests, I also added the collapsed option to the LayerControl since that was easier then getting playwright to hover the mouse and take a screenshot

**Comment by martinRenou** [comment]
> I added the UI tests

You probably forgot to commit the files for that? Only `leaflet.py` changed in the PR

**Comment by gjmooney** [comment]
> > I added the UI tests
> 
> You probably forgot to commit the files for that? Only `leaflet.py` changed in the PR

lol woops

**Comment by martinRenou** [comment]
update galata references

**Review by martinRenou — APPROVED** [review]
Thanks!

**Pull request merged**